### DISCUSSION TITLE
Contrubuting for GSOC '23

### DISCUSSION THREAD

**Discussion by kshitij01042002** [cncf/mentoring#754] [Q&A] [answered]
I am fairly new to Open-Source and want to be a mentee at CNCF for GSoC '23, soo how can I get started?

**Comment by caniszczyk** [comment, marked as answer]
See https://contribute.cncf.io and join the #mentoring slack channel on slack.cncf.io

You can see how previous mentorships went, e.g., https://www.cncf.io/blog/2022/12/08/congratulations-to-24-cncf-fall-term-lfx-program-mentees/

**Comment by kshitij01042002** [comment]
Okay! Thanks

**Comment by Harshit2929** [comment]
@nate-double-u @caniszczyk   I had a doubt that if a contributor participated in lfx mentorship program can that same contributor participate in gsoc too?. is it necessary that an open source contributor needs to be a beginner in open source to participate in gsoc?

**Reply by aliok** [reply]
There's some guidelines here: https://developers.google.com/open-source/gsoc/faq#can_someone_already_participating_in_open_source_be_a_gsoc_contributor

However, Google and CNCF GSoC admins will give the 👍 or 👎

**Comment by evershalik** [comment]
I am also interested in contributing to CNCF for GSoC'23.
